mount/mount_freebsd.go

```diff
@@ -0,0 +1,137 @@
+// +build freebsd
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package mount
+
+import (
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/pkg/errors"
+	"golang.org/x/sys/unix"
+)
+
+var (
+	// ErrNotImplementOnUnix is returned for methods that are not implemented
+	ErrNotImplementOnUnix = errors.New("not implemented under unix")
```

Comment:
1. You haven't used this constant on the PR.
2. Would it be `ErrNotImplemented` or `ErrNotImplementedOnPlatform`? This might be useful for Windows.

Reply:
This constant is pre-existing (was in `mount/mount_unix.go`) and used elsewhere in the codebase when compiled for FreeBSD (specifically in `pkg/os/mount_unix.go`).  I left the name as-is to avoid touching other places in the codebase.

Reply:
Oh I see. LGTM then.